contracts/GNTDeposit.sol

```diff
@@ -184,11 +218,35 @@ contract GNTDeposit is ReceivingContract, Ownable {
 
     function _reimburse(address _from, address _to, uint256 _amount) private {
         require(balances[_from] >= _amount);
+        if (daily_reimbursement_limit != 0) {
+            if (current_reimbursement_day != block.timestamp / 1 days) {
```

Comment:
unsure if solidity compiler optimizes that - but wouldn't it be better to calculate the value once? and only compare/assign it later? question 2 -> are those fixed point divisions? what's the type of the right-hand side value? also unit256?

Reply:
No, no, no, uint256, yes

Reply:
btw, another question - is there a chance for a race condition that could change the value of any compared values (both `block.timestamp` and the limits) while this block is executed?

Reply:
plus ... if those are not fixed point divisions - is it safe to check for equality?